framework/set/defaults/defaults.go

```diff
@@ -75,13 +78,60 @@ const (
 	VpcSecurityGroupIds      = "vpc_security_group_ids"
 	KeyName                  = "key_name"
 	AwsInstance              = "aws_instance"
-	Name                     = "Name"
+	Name                     = "name"
```

Comment:
Is this specific for Harvester? If so, can you introduce a separate name? This will inadvertently break a number of tests that rely on the specific `Name` style.

Reply:
lets discuss a name for this, since the capital N breaks the convention of all the other strings in defaults